oat-sa/extension-tao-qtiprint
# Feature/BSA-201/add table renderer

Related to: https://oat-sa.atlassian.net/browse/BSA-201

Added renderer for `qtiClass: 'table'`
Added css rules for table.

How to test:
1. install `wkhtmltopdf` https://hub.taotesting.com/techdocs/tao-booklet/readme-booklet
2. set `define('DEBUG_MODE', false);` in `config/generis.conf.php`
3. in `config/taoBooklet/wkhtmltopdf.conf.php` set binary to wkhtmltopdf path for example `'binary' => '/usr/local/bin/wkhtmltopdf',`
3. compile bundles for taoQtiPrint and taoBooklet
`cd tao/views/build`
`npx grunt taoqtiprintbundle` 
`npx grunt taobookletbundle` 
4. create shared stimulus **with table**
5. create items with shared stimulus
6. create test with these items
7. generate booklet from test
8. open booklet in preview and download pdf
9. booklet should render without errors
